core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -73,34 +75,40 @@ private static String prependTitleIfRequired(String icon, String title) {
         return icon;
     }
 
-    public static String getIonicon(String name, String title) {
```

Comment:
This is public API available to plugins…?

Reply:
not used anywhere though

Reply:
How do you know?

But really, could we perhaps be more careful when introducing new API to ensure we are only adding what we can commit to support longer term?

Reply:
Searched via code search and it’s not an API that would make sense for others to call.

I’ve gone through earlier and asked for lots of methods to be excluded from API, will double check again though

Reply:
This undocumented change fails PCT for `secure-requester-whitelist` thanks to its (luckily unnecessary) `icon-set` test dependency: https://github.com/jenkinsci/secure-requester-whitelist-plugin/pull/9